src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -1,44 +1,145 @@
-                "-O", output.getAbsolutePath()
+                "-I", tumorBam.getAbsolutePath(),
+                "-tumor", tumorName,
+                "-I", normalBam.getAbsolutePath(),
+                "-normal", normalName,
+                "-R", b37_reference_20_21,
+                "-dbsnp", dbsnp_138_b37_20_21_vcf,
+                "-L", "20:10000000-10100000", // this is 1/3 of the chr 20 interval of our mini-dbSNP
+                "-O", outputVcf.getAbsolutePath()
         };
 
         runCommandLine(args);
+        final long numVariants = StreamSupport.stream(new FeatureDataSource<VariantContext>(outputVcf).spliterator(), false).count();
+        Assert.assertTrue(numVariants < 4);
     }
 
+    // run tumor-only using our mini dbSNP on NA12878, which is not a tumor
+    // since any "somatic variants" are germline hets with good likelihoods, dbSNP doesn't actually do anything
+    // we're just making sure nothing blows up
+    @Test
+    public void testTumorOnlyWithDbsnp() throws Exception {
+        Utils.resetRandomGenerator();
+        final File outputVcf = createTempFile("output", ".vcf");
+
+        final String[] args = {
+                "-I", NA12878_20_21_WGS_bam,
+                "-tumor", "NA12878",
+                "-R", b37_reference_20_21,
+                "-dbsnp", dbsnp_138_b37_20_21_vcf,
+                "-L", "20:10000000-10010000",//"20:10000000-10300000",   // this is the chr 20 interval of our mini-dbSNP
+                "-O", outputVcf.getAbsolutePath()
+        };
+
+        runCommandLine(args);
+        final long numVariants = StreamSupport.stream(new FeatureDataSource<VariantContext>(outputVcf).spliterator(), false).count();
+    }
+
+    // tumor bam, tumor sample name, normal bam, normal sample name, truth vcf, required sensitivity
+    @DataProvider(name = "dreamSyntheticData")
+    public Object[][] dreamSyntheticData() {
+        return new Object[][]{
+                {new File(DREAM_BAMS_DIR, "tumor_1.bam"), "synthetic.challenge.set1.tumor", new File(DREAM_BAMS_DIR, "normal_1.bam"), "synthetic.challenge.set1.normal", new File(DREAM_VCFS_DIR, "sample_1.vcf"), 0.97},
+                {new File(DREAM_BAMS_DIR, "tumor_2.bam"), "background.synth.challenge2.snvs.svs.tumorbackground", new File(DREAM_BAMS_DIR, "normal_2.bam"), "synthetic.challenge.set2.normal", new File(DREAM_VCFS_DIR, "sample_2.vcf"), 0.97},
+                {new File(DREAM_BAMS_DIR, "tumor_3.bam"), "IS3.snv.indel.sv", new File(DREAM_BAMS_DIR, "normal_3.bam"), "G15512.prenormal.sorted", new File(DREAM_VCFS_DIR, "sample_3.vcf"), 0.93},
+                {new File(DREAM_BAMS_DIR, "tumor_4.bam"), "synthetic.challenge.set4.tumour", new File(DREAM_BAMS_DIR, "normal_4.bam"), "synthetic.challenge.set4.normal", new File(DREAM_VCFS_DIR, "sample_4.vcf"), 0.8}
+        };
+    }
+
+    //TODO: bring this to HaplotypeCallerIntegrationTest
+    private Pair<Double, Double> calculateConcordance(final File outputVcf, final File truthVcf ) {
```

Comment:
We should check whether the variant context in the output vcf is filtered. The output vcf in `testDreamTumorNormal` has one filtered site (20:38245275, alt_allele_in_normal) and it's counted as a false positive. This applies to `calculateConcordance` in `HaplotypeCallerIntegrationTest` also.